docs/articles_en/about-openvino/contributing.rst

```diff
@@ -0,0 +1,169 @@
+Contribute to OpenVINO
+========================
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   contributing/code-contribution-guide
+
+OpenVINO™ is always looking for opportunities to improve and your contributions
+play a big role in this process. Here are four ways you can make OpenVINO better:
+
+- `Provide feedback <#provide-feedback>`__
+- `Contribute code changes <#contribute-code-changes>`__
+- `Improve documentation <#improve-documentation>`__
+- `Promote and support OpenVINO <#promote-and-support-openvino>`__
+
+
+:fas:`comments` Provide feedback
+################################
+
+.. rubric:: Report bugs / issues
+   :name: report-bugs-issues
+
+If you notice unexpected behavior in OpenVINO or its components, you can
+`create a new issue <https://github.com/openvinotoolkit/openvino/issues>`__
+in the GitHub issue tracker.
+
+.. rubric:: Propose improvements
+   :name: propose-improvements
+
+If you want to share your ideas for improving OpenVINO:
+
+- Open a new `GitHub Discussion <https://github.com/openvinotoolkit/openvino/discussions>`__.
+- Create a `Feature Request Issue <https://github.com/openvinotoolkit/openvino/issues/new?assignees=octocat&labels=enhancement%2Cfeature&projects=&template=feature_request.yml&title=%5BFeature+Request%5D%3A+>`__
+  if your idea is already well defined.
+
+In both cases, provide a detailed description and list potential use cases,
+benefits, and challenges. Keep in mind that even if your input is not immediately
+prioritized, it may be used at a later or undertaken by the community.
+
+
+:fas:`code-branch` Contribute code changes
```

Comment:
GFI could also be mentioned here

Reply:
GFI is emphasized in the tip admonition below, line 60.

Reply:
![image](https://github.com/user-attachments/assets/e4df8aa5-f8ca-4f58-a070-a353cf368761)